doc/apiv3-documentation.apib

```diff
@@ -2139,6 +2139,84 @@ For more details and all possible responses see the general specification of [Fo
                 "message": "The request body was not empty."
             }
 
+
+# Group Revisions
+
+Revisions are sets of updates to files in the context of repositories linked in OpenProject.
+
+## Linked Properties
+|  Link            | Description                                         | Type          | Constraints | Supported operations |
+|:----------------:| --------------------------------------------------- | ------------- | ----------- | -------------------- |
+| self             | This revision                                       | Revision      | not null    | READ                 |
+| project          | The project to which the revision belongs           | Project       | not null    | READ                 |
+| committer        | The user that committed this revision, if defined   | User          |             | READ                 |
+
+
+## Local Properties
+| Property       | Description                                             | Type        | Constraints | Supported operations |
+|:--------------:| --------------------------------------------------------| ----------- | ----------- | -------------------- |
+| id             | Revisions's id                                          | Integer     | x > 0       | READ                 |
+| identifier     | The SCM identifier of the revision (e.g. SHA hash)      | String      | not null    | READ                 |
+| committerName  | The name of the author that committed this revision     | String      | not null    | READ                 |
+| message        | The commit message of the revision                      | Formattable | not null    | READ                 |
+| committedAt    | The time this revision was committed to the repository  | DateTime    | not null    | READ                 |
```

Comment:
I'd again vote for `createdAt`, as this is APIv3-speak we are using in all other resources too.

I would only choose other words if you wanted to differentiate between (in git semantics) [commit date and the author date](http://stackoverflow.com/questions/11856983/why-git-authordate-is-different-from-commitdate).

In this case I'd probably use `createdAt` (because this is the APIv3 way to say it) and `commitedAt` (for the commit date in git semantics).